nucleus/__init__.py

```diff
@@ -1177,3 +1179,52 @@ def _set_api_key(self, api_key):
             raise NoAPIKey()
 
         return api_key
+
+    def create_dataset_from_dir(
+        self,
+        dirname: str,
+        dataset_name: Optional[str] = None,
+        use_privacy_mode: bool = False,
+        privacy_mode_proxy: str = "",
+        file_ext_globs: Tuple[str] = ("png", "jpg", "jpeg"),
```

Comment:
I think I'd give a less technical name for this and hide the underlying implementation, after all, these are just file extensions, not globs. (Also I just encountered the term glob now for the first time. I know, it's embarrassing 🙈)

Reply:
Changed it to `allowed_file_types`